bitnami/thanos/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/thanos
-version: 12.8.3
+version: 12.8.4
```

In these changes we are adding a new feature, support for `dnsPolicy`. According to [semver](http://semver.org) we should bump a minor version:

```suggestion
version: 12.9.0
```

---

I bumped chart version to 12.9.0 as indicated.